src/dataMap.js

```diff
@@ -594,7 +594,7 @@ class DataMap {
       for (let i = 0, ilen = sliced.length; i < ilen; i++) {
         out = out[sliced[i]];
 
-        if (typeof out === 'undefined') {
+        if (out === undefined || out === null) {
```

Comment:
We prefer `void 0` over `undefined` so without `typeof` it should look like:

```suggestion
        if (out === void 0 || out === null) {
```

Thanks for the PR!